shared/src/commonMain/kotlin/ui/onboarding/OnboardingItems.kt

```diff
@@ -0,0 +1,42 @@
+package ui.onboarding
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import dev.icerock.moko.resources.StringResource
+
+internal class OnboardingItems(
+    val title: String,
+    val desc: String,
+    val extraItems: @Composable () -> Unit = {}
+) {
+    companion object{
+        fun getData(): List<OnboardingItems>{
+            return listOf(
+                OnboardingItems(
+                    title = "ABCD",
```

Comment:
I know you didn't write this code, but could you write a few onboarding texts? I don't have time at the moment

Reply:
sure, i would like you to proof read it tho

Reply:
The intellij squiggly line disappeared, so I assumed it was fine.